app/jobs/delegates_metadata_sync_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class DelegatesMetadataSyncJob < WcaCronjob
+  def perform
+    UserGroup.delegate_regions.flat_map(&:roles).map do |role|
+      unless role.is_lead?
+        user = role.user
+        first_delegated = user.actually_delegated_competitions.to_a.minimum(:start_date)
+        last_delegated = user.actually_delegated_competitions.to_a.maximum(:start_date)
+        total_delegated = user.actually_delegated_competitions.to_a.length
+        role.metadata.first_delegated = first_delegated
+        role.metadata.last_delegated = last_delegated
+        role.metadata.total_delegated = total_delegated
```

Comment:
Now you don't even need the variables anymore.
Instead of `foo = my_cool_method; role.metadata.first_delegated = foo;` why not just do `role.metadatafirst_delegated = my_cool_method`?

Reply:
Oh yes, done.